editor/editor_about.cpp

```diff
@@ -94,6 +94,7 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 			il->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 			il->set_same_column_width(true);
 			il->set_auto_height(true);
+            il->set_auto_width(false);
```

Comment:
You used spaces, but Godot uses tabs.

Reply:
oh! thank you!
That is something I am always forgetting to check for